packages/block-library/src/search/block.json

```
@@ -85,7 +85,10 @@
 				"width": true
 			}
 		},
-		"html": false
+		"html": false,
+		"interactivity": {
+			"clientNavigation": true
+		}
```

This one is already interactive.

```suggestion
		"html": false
```

---

Done